locale/es/LC_MESSAGES/django.po

```diff
@@ -216,7 +221,7 @@ msgstr "Cambiar idioma"
 
 #: who/templates/base.html:151
 msgid "WHO"
-msgstr ""
+msgstr "OMS"
```

Comment:
I don't know if we should be translating this :thinking:

Reply:
@KaitCrawford Sure we can do that makes sense.